x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/list_page/index.tsx

```diff
@@ -34,7 +34,8 @@ import {
   useUrlParams,
   useBreadcrumbs,
 } from '../../../hooks';
-import { AgentPolicySummaryLine, SearchBar } from '../../../components';
+import { SearchBar } from '../../../components';
```

Comment:
Why has this changed?

Reply:
When I tried to add jest tests for `AgentPolicyListPage`, I always got the error that _'react expected a component but got undefined'_, and a hint that I probably exported a component in a wrong way. So after a _lot_ of debugging I found that a component, namely the `AgentPolicySummaryLine` component was indeed `undefined`, and it turned out that it was imported from the wrong folder.

It is actually 5 levels higher:
```
x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/list_page/index.tsx
x-pack/plugins/fleet/public/components/link_and_revision.tsx
```

I don't know why it works in React environment with the wrong import, but - unlike in jest - it works somehow, here's a little test:

<img width="720" alt="image" src="https://user-images.githubusercontent.com/39014407/231854860-b92f0625-27b1-4707-bbb2-b354ac1b4120.png">



<img width="478" alt="image" src="https://user-images.githubusercontent.com/39014407/231854124-fcf9f38f-e3cf-49eb-b05b-6c70b239115a.png">

in the browser:
<img width="505" alt="image" src="https://user-images.githubusercontent.com/39014407/231854193-fefaa6d5-3c7e-4710-b5e0-084a9c94611f.png">